README.md

```diff
@@ -74,8 +86,12 @@ https://cn.dapr.io/meeting/
 
 Every week, we meet to plan and sync on the progress for the milestone's release. This meeting includes maintainers and active contributors and usually lasts for about 30-60 minutes. If you would like to **actively contribute** towards a milestone, you are welcome to join that meeting too.
 
-**Zoom Call:** https://zoom.us/j/92337656846<br>
-**Passcode:** 815623<br>
+<!--
+We are using a different link from Community Call because the Community Call requires the host to admit each person. In the weekly syncs, we "never" login with the host account.
+-->
+**Zoom Call:** https://zoom.us/j/91940016938<br> <!-- DO NOT ADD PASSWORD TO THE MEETING URL TO AVOID ZOOM-BOMBING (yes, we had that before). -->
+**Meeting ID:** 919 4001 6938<br>
+**Passcode:** 152697<br> <!-- If we get Zoom-bombing, generate a new one and only share on request, not here>
```

Comment:
We have restriction on the call itself to prevent this, so I really would not fixate on the passcode here. The restrictions are better with having a muted callers by default and wait room etc. 
```suggestion
**Passcode:** 152697<br> 
```